### PR TITLE
Pagination component

### LINKED ISSUE #53: Pagination component
Screen Shot 2021-08-15 at 9 50 33 PM

### PR DESCRIPTION
**Pull request by panjenny0**
## Description
Created the pagination component used in the resources section. Closes #53 
### interactions
ezgif com-gif-maker (2)
### responsiveness
- shrinks with other text when screen size is reduced
ezgif com-gif-maker (1)

### CODE REVIEW

File: components/Pagination.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+const BasePaginationButton = styled.button`
+  ${(p) =>
+    p.contentUrl &&
+    `&:before {
+        display: inline-block;
+        height: 24px;
+        line-height: normal;
+        content: url(${p.contentUrl});
+    }`}
+  font-family: HK Grotesk;
```
Comment (kevin-zou) [inline components/Pagination.js:16]: NIt: do we need this if we defined it as our global font?

**Review by kevin-zou — APPROVED** [review]
Clean PR 😎  Small nit pick but this looks great!

File: components/Pagination.js (hunk 1 above)
Comment (panjenny0) [reply components/Pagination.js:16]: Thanks for the speedy feedback!
I tried not specifying the `font-family`, but for some reason, the font for the `button` component defaulted to Arial, so I had to specify HK Grotesk
Screen Shot 2021-08-20 at 1 06 51 PM

Comment (kevin-zou) [reply components/Pagination.js:16]: Oh that's weird, thanks for the response 🚀 Lgtm!

**Pull request merged**